internal/instance/resource_instance_test.go

```diff
@@ -1076,6 +1076,67 @@ func TestAccInstance_waitForIPv4AndIPv6(t *testing.T) {
 	})
 }
 
+func TestAccInstance_containerRename(t *testing.T) {
+	instanceName := petname.Generate(2, "-")
+	newInstanceName := petname.Generate(2, "-")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccInstance_container(instanceName),
```

Comment:
I would suggest to add a comment here that explain why this is working (the resource name is always `instance1`)